### PR TITLE
Fix font_size default. add r2rtf to Suggests. vbump. NEWS

### PR DESCRIPTION
**Pull request by gmbecker**
[skip vbump]

### CODE REVIEW

**Comment by shajoezhu** [comment]
Hi @gmbecker , I would suggest to go in after #636 and #607

**Comment by gmbecker** [comment]
> Hi @gmbecker , I would suggest to go in after #636 and #607 , we can better plan this and downstream testing during the next PI

If this were a different change I would agree but we don't want to make breaking changes to default font_size twice in a row. We are already committed to unifying font_size as having a default of 8, so we really should put this in now and tear the bandaid off. @shajoezhu

File: R/tt_export.R [hunk 1]
```diff
@@ -504,86 +504,3 @@ export_as_pdf <- function(tt,
     }
      list(file = file, npages = npages, exceeds_width = exceeds_width, exceeds_height = exceeds_height, lpp = lpp, cpp = cpp)
 }
-
-.margin_lines_to_in <- function(margins, font_size, font_family) {
```
Comment (shajoezhu) [inline R/tt_export.R:508]: hi @gmbecker , this was part of the migration right?

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks @gmbecker

File: R/tt_export.R (hunk 1 above)
Comment (gmbecker) [reply R/tt_export.R:508]: Yes, its no longer needed because we are now interpreting margins as inches values directly.

**Pull request merged**